awx/main/models/inventory.py

```diff
@@ -2364,7 +2370,7 @@ def _compat_compose_vars(self):
             'gce_image': 'image',
             # We need this as long as hostnames is non-default, otherwise hosts
             # will not be addressed correctly, was returned in script
-            'ansible_ssh_host': 'networkInterfaces[0].accessConfigs[0].natIP'
+            'ansible_ssh_host': 'networkInterfaces[0].accessConfigs[0].natIP | default(networkInterfaces[0].networkIP)'
```

Comment:
Is there a way to fall back to `None` in cause that also is not there?

Reply:
Yes, there is, but that's not really what the issue was asking for. I'm also not sure when and if the private IP could ever be none.